schemas/context/person.schema.json

```diff
@@ -56,6 +56,27 @@
                     },
                     "description": "Gender identity of the person.\n",
                     "default": "not_specified"
+                },
+                "xdm:maritalStatus": {
+                    "title": "Marital Status",
+                    "type": "string",
+                    "enum": [
+                        "married",
+                        "single",
+                        "not_specified"
+                    ],
+                    "meta:enum": {
```

Comment:
these are the only enums we are going to specify?  Not `divorced`, `widowed`, etc?

Reply:
Valid point.  I was trying to capture this as its showing up on a customer schema and wanted to ensure we capture it as well.  I can add other values here if we think this is relevant to add as a field at all.  Thoughts?

Reply:
I am fine with it as a new field in the schema, but you need to make sure that when using an enum, you specify all the correct values - as you can't add them later.

Reply:
Resolved in latest commit